### DISCUSSION TITLE
Mode-line keep changing when typing or moving cursor

### DISCUSSION THREAD

**Discussion by allenryb** [joaotavora/eglot#784] [Q&A] [answered]
I am new from elpy to eglot. 
I use `eglot + pylsp`.
When I type or move cursor, the number in the following RED bracket keeps changing and it is annoying. 

![modeline](https://user-images.githubusercontent.com/6965371/148635491-7f89cbe0-6b30-441c-8b42-91f78cb4c479.png)

What does this number mean? How to disable it?

**Comment by joaotavora** [comment, marked as answer]
Ah, yes. That is the number of outgoing, still unanswered LSP requests to
the server. It's useful for knowing how laggy/responsive a server is. Kind
of a debugging feature. I guess we could hide it, or at least only show it
when it exceeds a certain amount...

**Reply by nemethf** [reply]
Something like the patch bellow might help to understand what's going on.

```diff --git a/eglot.el b/eglot.el
index a6c60e9..6c5aa4e 100644
--- a/eglot.el
+++ b/eglot.el
@@ -1668,7 +1668,8 @@ Uses THING, FACE, DEFS and PREPEND."
              `("/" ,(eglot--mode-line-props
                      (format "%d" pending) 'warning
                      '((mouse-3 eglot-forget-pending-continuations
-                                "forget pending continuations"))))))))))
+                                "forget pending continuations"))
+                     "Number of outgoing, still unanswered LSP requests to the server"))))))))
 
 (add-to-list 'mode-line-misc-info
              `(eglot--managed-mode (" [" eglot--mode-line-format "] ")))
```

**Reply by skangas** [reply]
I guess this shows a tooltip? If so, LGTM.

**Reply by joaotavora** [reply]
LGTM to me too.

**Reply by skangas** [reply]
@nemethf Could you please install this change on the master branch? I'd like to have it in version 1.8 if possible (see #782).
